shell/platform/darwin/ios/framework/Headers/FlutterEngine.h

```diff
@@ -14,6 +14,8 @@
 #include "FlutterPlugin.h"
 #include "FlutterTexture.h"
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
Move `NS_ASSUME_NONNULL_BEGIN` to after the `@class` forward declaration.  I'm not actually sure if it matters but that seems to be the pattern?  Whatever you think.

Reply:
Sure, I have no preference.